NSS-Day-Cohort-42/rare-media-moguls
# Category Buttons List rendered to home page.

#### Changes Made
1. Added file `CategoryButton.js`, `CategoryButtonList.js`, and `Category.css` files to `categories` directory.
1. Modified `ApplicationViews.js` to include `CategoryButtonList` on path routed to `/`.

#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout mp-categories
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality. 
> When user is logged in and on the 'home' page [route path `/`], a list of clickable category buttons should be rendered on the right side of the screen. 
> When user selects a category from the list, the `selectedCategoryId` variable is set to the value of the chosen category id.

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
> NOTE: The commented out useEffect in CategoryButtonList is for future use when filtering posts by category id
